plugins/Live/Model.php

```diff
@@ -395,7 +395,7 @@ public function makeLogVisitsQueryString($idSite, $period, $date, $segment, $off
         $groupBy = false;
         $limit = $limit >= 1 ? (int)$limit : 0;
         $offset = $offset >= 1 ? (int)$offset : 0;
-        $orderBy = "idsite, visit_last_action_time " . $filterSortOrder;
+        $orderBy = "visit_last_action_time " . $filterSortOrder;
```

Comment:
I think this is unrelated to this change. Can you issue a different PR for this?